task/idle/Cargo.toml

```diff
@@ -15,7 +15,7 @@ userlib = {path = "../../sys/userlib"}
 [target.'cfg(target_arch = "arm")'.dependencies]
 cortex-m = { version = "0.7", features = ["inline-asm"] }
 
-[target.'cfg(target_arch = "riscv32")'.dependencies]
+[target.'cfg(any(target_arch = "riscv32", target_arch = "riscv64"))'.dependencies]
```

Comment:
I don't think idle actually uses the `riscv` crate.

Reply:
ACK. I'll try removing this entire block. This change was purely to put it on the same footing as riscv32.